src/main/java/seedu/address/model/country/Country.java

```diff
@@ -23,6 +26,15 @@ public Country(String countryCode) {
         this.countryName = new Locale("", countryCode).getDisplayName();
     }
 
+    private Country() {
+        this.countryCode = NONE_COUNTRY_CODE;
+        this.countryName = NONE_COUNTRY_NAME;
+    }
+
+    public static Country getNullCountry() {
+        return new Country();
+    }
```

Comment:
Suggestion: have a public static NULL_COUNTRY object = new Country();, and just use Country.NULL_COUNTRY to access it.
This is what I did for my ContractExpiryDate, so you're not creating new objects all the time, and imo it makes it clearer to the reader.

Reply:
Yup, good idea!